platform-ajax/src/test/java/com/softicar/platform/ajax/drag/and/drop/AjaxDragAndDropTest.java

```diff
@@ -43,7 +43,7 @@ public TestDiv() {
 			this.dragTargetDiv = appendChild(new Div(200, 100));
 			this.dragAnchorDiv = dragTargetDiv.appendChild(new Div(50, 50));
 
-			getDomEngine().makeDraggable(dragTargetDiv, dragAnchorDiv);
+			getDomEngine().makeDraggable(dragTargetDiv, dragAnchorDiv, null);
```

Comment:
it seems that this `null` value is not properly converted to a `null` value in JavaScript, we should try to fix this code in `DomAjaxEngine`. At least `AjaxDragAndDropTest` passes when change to plain `"null"`.
![image](https://user-images.githubusercontent.com/83839745/168895195-d44753f1-5d0b-42cd-9d9b-94a7fc960303.png)

Okay, I changed this locally and all check pass.

Reply:
Seems to work, thanks for the proposal